Umbraco-Uno/Working-With-Forms/Setting-up-a-form/index.md

```diff
@@ -0,0 +1,46 @@
+---
+versionFrom: 8.0.0
+---
+
+# Setting up a form
+
+In this guide, you will learn how to set up a form in the Umbraco Uno backoffice. 
+
+When you get your Uno project, it will not have any forms, so you will have to make them your self which can be a challenge when you have never done it before. In this guide, you will learn how to build a form from scratch.
+If this is your first form, you will have a green button in the middle of the screen that says ***Create form*** if this is not your first form, proceed to the next steps.
+
+* Navigate to the Forms section in the backoffice.
+* In the tree on the left, hover ***Forms*** and select the ***...***
+* On the list that shows up, select "Empty form."
+
+![This image shows the forms section and where to press for a new form](images/Forms-backoffice.png)
+
+## Building the form
+
+Now we have taken the first step to creating our very own form. In the next steps, we will name the form and add the fields that it needs to contain.
+
+![An image showing the main view of the new form](images/Back-form.png)
+
+:::note
+You will see that the form already contains an item. This is the permission to store data, so if you need this for the form you are creating, you can go ahead and leave that. Otherwise, you can delete it by clicking the bin icon at the top of the field.
+:::
+
+* Give the new form a name where it says ***Enter Name***.
+* Now to adding a group Click where it says ***Add new group***.
```

Comment:
You will only have this option if you chose to remove the "Consent" field, right?

```suggestion
* Now to add a group click where it says ***Add new group***.
```

Reply:
i believe you have it no matter what

Reply:
Ah, OK! So, the guide shows how to create fields that are in another Group than the Consent field.
I missed that.